### PR TITLE
list of fruits added

### PR DESCRIPTION
**Pull request by etech-staff**
please review and advise/approve

### CODE REVIEW

File: file1 [hunk 1]
```diff
@@ -1,2 +1,9 @@
 we are enjoying git
 yet another line
+
```
Comment (etechDevops) [inline file1:3]: code looks great

**Review by etechDevops — APPROVED** [review]

File: file1 [hunk 2]
```diff
@@ -1,2 +1,9 @@
...
+List of fruits
```
Comment (etechDevops) [inline file1:4]: great job!

**Review by etechDevops — APPROVED** [review]
awesome!

File: file1 [hunk 3]
```diff
@@ -1,2 +1,9 @@
 we are enjoying git
```
Comment (etech-staff) [inline file1:1]: great

File: file1 (hunk 1 above)
Comment (etech-staff) [inline file1:3]: awesome

File: file1 [hunk 4]
```diff
@@ -1,2 +1,9 @@
...
 yet another line
+
+List of fruits
+Apple
```
Comment (etech-staff) [inline file1:5]: good!

File: file1 [hunk 5]
```diff
@@ -1,2 +1,9 @@
...
+Banana
+Orange
+Mango
```
Comment (etech-staff) [inline file1:8]: great!

**Review by etech-staff — COMMENTED** [review]
Thank you

**Review by rnfor-pro — COMMENTED** [review]
great

File: file1 (hunk 1 above)
Comment (rnfor-pro) [inline file1:3]: great

**Review by rnfor-pro — COMMENTED** [review]
awesome

**Pull request merged**